server/settings/main.py

```diff
@@ -1,67 +1,180 @@
-from pydantic import Field
+from ayon_server.entities.core.attrib import attribute_library
+from ayon_server.settings import BaseSettingsModel, SettingsField
+from ayon_server.settings.enum import secrets_enum, anatomy_presets_enum
 
-from ayon_server.settings import BaseSettingsModel
-from ayon_server.settings.enum import secrets_enum
-from ayon_server.graphql.resolvers.users import get_users
+
+def default_shotgrid_entities():
```

Comment:
please split these in two different functions, one should be the types that you can enable and the other one the ones that are selected by default:
```
def get_shotgrid_entities():
    """The entity types that exist in ShotGrid."""
    return [
        "Project",
        "Episode",
        "Sequence",
        "Scene",
        "Shot",
        "Asset",
        "Task",
        "Version",
    ]


def get_default_shotgrid_enabled_entities():
    """The entity types in Shotgrid that are enabled by default in AYON."""
    return [
        "Project",
        "Episode",
        "Sequence",
        "Shot",
        "Asset",
        "Task",
        "Version",
    ]
```

Reply:
or without the `get` if it's the convention for enums

Reply:
Yeah, I was actually going to do it, thanks for saving me time :D